src/go/rpk/pkg/cli/cmd/cluster/config/export/export.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package export
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sort"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/out"
+	yaml "gopkg.in/yaml.v3"
+)
+
+func ExportConfig(
+	file *os.File, schema admin.ConfigSchema, config admin.Config, all bool,
+) (err error) {
+	template := make(map[string]interface{})
+
+	// Present properties in alphabetical order, providing some pseudo-grouping based on common prefixes
+	keys := make([]string, 0, len(schema))
+	for k := range schema {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+
+	yamlNode := yaml.Node{
+		Kind: yaml.MappingNode,
+	}
+
+	for _, name := range keys {
+		meta := schema[name]
+		curValue := config[name]
+		template[name] = curValue
+
+		visibility := meta.Visibility
+
+		if visibility == "deprecated" {
+			continue
+		}
+
+		if meta.Visibility == "tunable" && !all {
+			continue
+		}
+
+		property_node := yaml.Node{
+			Kind:  yaml.ScalarNode,
+			Value: name,
+		}
+
+		var valueNode yaml.Node
+		switch x := curValue.(type) {
+		case int:
+			valueNode = yaml.Node{
+				Kind:  yaml.ScalarNode,
+				Value: fmt.Sprintf("%d", x),
```

Comment:
`strconv.Atoi` is a bit clearer, here and below

Reply:
Converted these to `strconv` equivalents.